third_party/bazel_rules/rules_kotlin/kotlin/native/wasm.bzl

```diff
@@ -1,3 +1,10 @@
 # Alias wasm split transition rule unless G3 Kotlin Native synced with GH rules
 def wasm_kt_binary(name, kt_target):
-    native.alias(name = name, actual = kt_target, visibility = ["//visibility:public"])
+    native.genrule(
+        name = name,
+        srcs = [kt_target + ".wasm", kt_target + ".wasm.js"],
+        cmd = "cp $(location %s.wasm) $(@D)/%s.wasm; cp $(location %s.wasm.js) $(@D)/%s.wasm.js" % (kt_target, name, kt_target, name),
```

Comment:
Can you add some comment on what is the purpose of this, so that future reader has more context?

Reply:
Basically, internally auto_wasm (wasm) creates another blaze build target that lives under a different blaze bin directory, like blaze-bin/test_particle_wasm/test.wasm . and the GH version used a alias() previously so that this special directory wasnt  created so that this file would live under bazel-bin/test_partible/test.wasm.    This script makes them both live under test_particle_wasm/ directory. This will allow me to make arcs_ts_test work on G3 and GH without messing with the particle load paths.